web/docs/tql2/operators/pipeline/list.md

```diff
@@ -0,0 +1,61 @@
+# list
+
+Shows managed pipelines.
+
+```tql
+package::list
+```
+
+## Description
+
+The `pipeline::list` operator returns the list of all managed pipelines.
+
+Managed pipelines are pipelines created through the [`/pipeline` API](/api).
+Notably, pipelines run through a `tenzir` process are not known to the node.
```

Comment:
What does this sentence mean? I struggle with understanding it after reading the sentence before.

Reply:
I'll just remove the sentence, it's irrelevant for most users.